wiki/Installing-Oppia-Android.md

```diff
@@ -81,38 +84,59 @@ Please follow these steps to set up Oppia Android on your local machine.
 ## Set up and run tests
 Testing the app is an integral part of our development process. You will need to test all code changes to ensure that the app works correctly, therefore it is important to ensure that your test configuration works.
 
-We run tests in either Espresso(`app` module tests) or Robolectric(non-app module tests), meaning tests will run in either the emulator or on Gradle/Bazel via the terminal.
+We strongly recommend running tests on Robolectric which is faster because it does not require a physical device or emulator setup.
+
+### Configure Robolectric Tests
+**Step-by-Step guidance for setting up and running app modules robolectric test:**
+
+1. Go to **Edit Configuration** in Android Studio (Bumblebee | 2021.1.1 Patch 3)
+   ![](https://user-images.githubusercontent.com/9396084/79109714-83525980-7d96-11ea-99d7-f83ea81a8a50.png)
+
+2. Click on Add(+) -> **JUnit**
+   ![](https://github.com/oppia/oppia-android/assets/76530270/87caf3fc-37d9-472d-92fd-b8ec49fb6b49)
+
+3. Enter following information:
+   - a) Name of test. Example: In my case "SplashActivityTest"
+   - b) Make sure select "java 11" and oppia-android.app
+   - c) Class path of Test class. Example: In my case "org.oppia.android.app.splash.
+   SplashActivityTest"
+   - d) Press `OK` to select the test.
+   ![](https://github.com/oppia/oppia-android/assets/76530270/5901624a-df76-4b27-8f31-6077a68fcb89)
+
+4. Click on "Run" button to run robolectric test. (In my case "SplashActivityTest")
+   ![](https://github.com/oppia/oppia-android/assets/76530270/75a6b998-90c5-4f0a-8886-78f96970be90)
+
+These are tests that are in non-app modules, such as **domain** or **utility**.
+1. In Android Studio, open the desired test file, e.g., `AnalyticsControllerTest`.
+2. In the test file, to the left of the class name, click on the orange and green arrow, and select **Run 'AnalyticsControllerTest'**.
+   - You will notice that the emulator is greyed out, but the run window will open to show the running tests:
+   - ![](https://user-images.githubusercontent.com/59600948/272657015-158117e5-47d2-40fc-a38b-5dee6c347556.png)
+
+**Espresso is slower for running tests, so we recommend using Robolectric.**
```

Comment:
I think this needs to go to the next line right below "### Configure Emulator Tests"

Reply:
Done.